maker/src/actor_system.rs

```diff
@@ -179,7 +181,9 @@ where
             .create(None)
             .spawn(&mut tasks);
 
-        tasks.add(time_to_first_position_ctx.run(time_to_first_position::Actor::new(db)));
+        tasks.add(time_to_first_position_ctx.run(time_to_first_position::Actor::new(db.clone())));
+
+        let _cull_old_dlcs_actor = cull_old_dlcs::Actor::new(db).create(None).spawn(&mut tasks);
```

Comment:
I wonder if this will make troubles as it's being started at the same time as other actors who are reading from the db. 

What do you think of running this once before we initialize anything else?

Reply:
> I wonder if this will make troubles as it's being started at the same time as other actors who are reading from the db.

I don't see potential for problems: we are just deleting irrelevant data that we never access otherwise. And I expect `sqlx` and `sqlite` to let us modify rows safely.

> What do you think of running cull once before we initialize anything else?

I see the appeal, but I'm not sure how to do it in a non-hacky way (i.e. with a long sleep before we do anything else in the actor system constructors).

Also we have to consider that after a few restarts most of the old DLC-related events will be culled already. It's only the very first time that we introduce this change that we may benefit significantly from running it first in isolation.

Reply:
We could just call it directly and await the result? 
As you said, the main benefit is for the first start, sub-sequential starts don't matter much which also means, that this call will be fast.

Reply:
> We could just call it directly and await the result? As you said, the main benefit is for the first start, sub-sequential starts don't matter much which also means, that this call will be fast.

Good idea! I've added https://github.com/itchysats/itchysats/pull/2041/commits/548cb5ff824ac0522676c27d86078fa18a8182ca.